bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/OpenhabGraalJSScriptEngine.java

```diff
@@ -135,6 +135,7 @@ public class OpenhabGraalJSScriptEngine
 
     // these fields start as null because they are populated on first use
     private @Nullable Consumer<String> scriptDependencyListener;
+    private String engineIdentifier;
```

Comment:
This seems unused? Only set, but never read.

Reply:
Yeah, like it was before (hence I removed it in a previous PR), but it proves very useful when debugging because it allows to identify the script engines …
I’d therefore like to have it

Reply:
It generates a SAT warning: `UnusedPrivateField`. Someone will remove it again in the future because of this, just like you recently did. Can you find a way to suppress the SAT warning and at the same time document in the code why it should be there?

Reply:
@jlaur Done 👍